### PR TITLE
Test TestCalcNetworkLosses

### PR DESCRIPTION
**Pull request by br4e**
- Scale by 1e18
- Fix loss initialization logic
- Fix naive/combined variable switch
- test

### CODE REVIEW

**Comment by br4e** [comment]
Should be good to go!

File: x/emissions/keeper/inference_synthesis/network_losses.go [hunk 1]
```diff
@@ -112,6 +112,15 @@ func CalcNetworkLosses(
 			if err != nil {
 				return emissions.ValueBundle{}, err
 			}
+			oneE18, err := alloraMath.NewDecFromString("1000000000000000000")
```
Comment (kpeluso) [inline x/emissions/keeper/inference_synthesis/network_losses.go:115]: should be a constant that gets imported from somewhere so you don't have to redefine each time

**Review by kpeluso — APPROVED** [review]
we may have to add more arithmetic with 1e18 other places e.g. in rewards

**Comment by br4e** [comment]
yeah we can figure out a longer term solution

**Pull request merged**